modules/nw-metallb-configure-bgp-advertisement.adoc

```diff
@@ -0,0 +1,143 @@
+    - fc00:f853:ccd:e799::/124
+  avoidBuggyIPs: true
+----
+
+.. Apply the configuration for the address pool:
++
+[source,terminal]
+----
+$ oc apply -f addresspool.yaml
+----
+
+. Create a BGPAdvertisement.
+
+.. Create a file, such as `bgpadvertisement.yaml`, with content like the following example:
++
+[source,yaml]
+----
+    apiVersion: metallb.io/v1beta1
+    kind: BGPAdvertisement
+    metadata:
+      name: bgpadvertisement
+      namespace: metallb-system
+    spec:
+      ipAddressPools:
+      - doc-example-bgp-basic
+----
+
+.. Apply the configuration:
++
+[source,terminal]
+----
+$ oc apply -f bgpadvertisement.yaml
+----
+
+== Example: Advertise an advanced address pool configuration with BGP
+
+In the following example, MetalLB assigns IP addresses to load-balancer services in the ranges between `203.0.113.200` and `203.0.113.203` and between `fc00:f853:ccd:e799::0` and `fc00:f853:ccd:e799::f`.
+
+To explain the two BGP advertisements, consider an instance when MetalLB assigns the IP address of `203.0.113.200` to a service.
+With that IP address as an example, the speaker advertises two routes to BGP peers:
+
+* `203.0.113.200/32`, with `localPref` set to `100` and the community set to the numeric value of the well-known `NO_ADVERTISE` community.
+This specification indicates to the peer routers that they can use this route but they should not propagate information about this route to BGP peers.
+
+* `203.0.113.200/30`, aggregates the load-balancer IP addresses assigned by MetalLB into a single route.
+MetalLB advertises the aggregated route to BGP peers with the community attribute set to `8000:800`.
+BGP peers propagate the `203.0.113.200/30` route to other BGP peers.
+When traffic is routed to a node with a speaker, the `203.0.113.200/32` route is used to forward the traffic into the cluster and to a pod that is associated with the service.
+
+As you add more services and MetalLB assigns more load-balancer IP addresses from the pool, peer routers receive one local route, `203.0.113.20x/32`, for each service, as well as the `203.0.113.200/30` aggregate route.
+Each service that you add generates the `/30` route, but MetalLB deduplicates the routes to one BGP advertisement before communicating with peer routers.
+
+.Procedure
+
+. Create an IPAddressPool.
+
+.. Create a file, such as `addresspool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1alpha1
+kind: IPAddressPool
+metadata:
+  namespace: metallb-system
+  name: doc-example-bgp-adv
+spec:
+  addresses:
+    - 203.0.113.200/30
+    - fc00:f853:ccd:e799::/124
+  autoAssign: false
+  avoidBuggyIPs: true
+----
+
+.. Apply the configuration for the address pool:
++
+[source,terminal]
+----
+$ oc apply -f addresspool.yaml
+----
+
+. Create a BGPAdvertisement.
+
+.. Create a file, such as `bgpadvertisement.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
```

Comment:
you need two bgpadvertisement instance to implement this.
there's only one aggregationlenght / localpref field.